connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -166,6 +170,23 @@ public void destroyConnector(final @PathParam("connector") String connector) thr
         completeOrForwardRequest(cb, "/connectors/" + connector, "DELETE", null);
     }
 
+    @GET
+    @Path("{connectorType}/config")
+    public ConfigDefInfo getConfigDef(final @PathParam("connectorType") String connectorType) throws Throwable {
+        FutureCallback<ConfigDefInfo> cb = new FutureCallback<>();
+        herder.getConfigDef(connectorType, cb);
+        return completeOrForwardRequest(cb, "/connectors/" + connectorType + "/config", "GET", null, new TypeReference<ConfigDefInfo>(){});
```

Comment:
Does it ever make sense to forward this? Requests that can be resolved on any node shouldn't be forwarded.